uploadPhoto.py

```diff
@@ -0,0 +1,45 @@
+import cv2
+import numpy as np
+import json
+import requests
+from datetime import datetime
+
+
+headers = {
+    "Authorization": "Bearer ##secret key##"}
+img_dir = "./photos/"
+
+
+def save_img_to_drive(img_name):
+    global headers
+    global img_dir
+    para = {
+        "name": img_name,
+        "parents": ["1iPsJb27oM8SQFLPf_E2tMiDEiac-9Caz"]
+    }
+    files = {
+        'data': ('metadata', json.dumps(para), 'application/json; charset=UTF-8'),
+        'file': open(img_dir+img_name, "rb")
+    }
+    r = requests.post(
+        "https://www.googleapis.com/upload/drive/v3/files?uploadType=multipart",
+        headers=headers,
+        files=files
+    )
+    print(r.text)
+
+
+cap = cv2.VideoCapture(0)  # fill rtsp
+# example 'rtsp://username:password@192.168.1.64/1'
+
+ret, frame = cap.read()
+if not ret:
+    print("No camera return")
+else:
+    img_name = datetime.now().strftime("%Y_%b_%d_%H_%M") + ".png"
+    cv2.imwrite(img_dir+img_name, frame)
```

Comment:
je opravdu potřeba fotku ukládat, když se stejně jen uploaduje? To nejde poslat přímo z paměti?. Když už tak to aspoň pak maž, abychom potom zbytečně nezaplnili kartu

Reply:
neni

Reply:
ale když nebude fungovat připojení na google tak o všechno přijdem

Reply:
No to podle mě tak kritické není :D, zase se mi to nechce co 2dny čistit. Imho pokud to chceš udělat takhle, tak by bylo ideální udělat další skript, který by to spouštělo a ten by bral jednotlivé fotografie a cyklem se snažil data posílat a při vyprázdnění se prostě zabil.

Reply:
SD karta má 64 GB nebudu hamoun vezmu si 30GB, za den naplním 1 GB z 2 kamer kdy udělám každých 10 minut 2 fotky = mám 30 dní 30 > 2....

Reply:
ok tak to ale aspoň, zkus poslat a až pak popřípadě uložit, nechceme pak zbytečně porovnávat a hledat které fotky se uploadnuly a které ne.